### ISSUE TITLE
Wrong Proposal Open Contract response structure

### ISSUE THREAD

**Issue by alvaxet** [binary-com/websockets#27]
This is about Version 3.

According to documentation on the left side here: https://developers.binary.com/api/#proposal_open_contract

response should include fields like entry_tick, entry_tick_time, high_barrier etc.

When I test it on that page or using websocket client I get this response structure:
```json
{  
   "echo_req":{  
      "proposal_open_contract":1
   },
   "proposal_open_contract":{  
      "barrier":"1.08675",
      "contract_id":"5749641848",
      "date_settlement":1447286399,
      "currency":"USD",
      "underlying":"frxEURUSD",
      "current_spot":"1.07763",
      "date_start":1446803742,
      "id":"a7b381c1f2ea11b7fec2a5ad3c5e5912",
      "is_intraday":0,
      "is_expired":0,
      "is_valid_to_sell":1,
      "shortcode":"CALL_FRXEURUSD_1.91_1446803742_1447286399F_S0P_0",
      "bid_price":"0.17",
      "is_forward_starting":0,
      "longcode":"USD 1.91 payout if EUR/USD is strictly higher than entry spot at close on 2015-11-11.",
      "date_expiry":1447286399,
      "is_path_dependent":0,
      "ask_price":"0.39",
      "current_spot_time":"1447075015",
      "payout":"1.91"
   },
   "msg_type":"proposal_open_contract"
}
```
So it seems like response doesn't include fields it supposed to include.

**Comment by raunakkathuria** [comment]
@alvaxet `entry_tick, entry_tick_time` will be returned only when contract is tick contract for example if you send this as proposal and purchase the contract then entry tick will be returned
```
{
  "proposal": 1,
  "amount": 10,
  "basis": "payout",
  "contract_type": "PUT",
  "currency": "USD",
  "symbol": "R_100",
  "duration": 5,
  "duration_unit": "t"
}
```

 and `high_barrier` will be returned only when you send contract that accepts two barrier, for example if you purchase following proposal and request proposal open contract then you will `high_barrier`

```
{
  "proposal": 1,
  "amount": 10,
  "basis": "payout",
  "contract_type": "EXPIRYRANGE",
  "currency": "USD",
  "symbol": "R_100",
  "duration": 5,
  "duration_unit": "m",
  "barrier": "+37.04",
  "barrier2": "-36.89"
}
```

**Comment by alvaxet** [comment]
So there is no way to get entry spot for all open contracts?

That is what I was originally asking about: https://github.com/binary-com/websockets/issues/11

In that ticket I've been told that it's done in "proposal_open_contract call". I've checked, and it doesn't have what I was asking about. 

I've reported about that and now it turns out that it's something completely different.

I need to get entry_spot/entry_price/entry_tick for all contracts I have in portfolio. 
How can I do that?
